web/vibe/web/rest.d

```diff
@@ -1603,8 +1604,8 @@ private HTTPServerRequestDelegate jsonMethodHandler(alias Func, size_t ridx, T)(
 		{
 			handleCors();
 			foreach (i, P; PTypes) {
-				static if (sroute.parameters[i].isOut) {
-					static assert (sroute.parameters[i].kind == ParameterKind.header);
+				static if (sroute.parameters[i].isOut &&
+					sroute.parameters[i].kind == ParameterKind.header) {
```

Comment:
Why ?

Reply:
https://github.com/vibe-d/vibe.d/blob/f1554380fefa186298d930eaeccf6f439924ff47/web/vibe/web/internal/rest/common.d#L298-L306

`in`parameters that are passed by reference are marked as `isOut`and end up in the body not header.

Reply:
Maybe fix that bug then ? :D

Reply:
I am not sure if that can be fixed easily. `const ref` and `in` are indistinguishable by looking at their storage class.

Reply:
But since REST only allows `const ref` but not `ref` they cant really be outputs actually.

https://github.com/vibe-d/vibe.d/pull/2495

Reply:
> are indistinguishable by looking at their storage class.

They are, using `ParameterStorageClass`. But it requires a new-ish release (2.094 IIRC)